feat: added basic event schemas

pethersonmoreno opened the pull request:
As suggestion of @halkeye in the Merge Request of gatsby-theme-nodeschool:
 - https://github.com/halkeye/gatsby-theme-nodeschool/pull/12/files/6324a96d076e80321dbf988583cea583b2c8b6a6

Moved schemas to gatsby-source-meetup as a Merge Request.

These changes can solve the issue: https://github.com/phacks/gatsby-source-meetup/issues/10

halkeye commented:
@phacks doesn't watch the repo, so its worth tagging him as well

phacks commented:
@pethersonmoreno @halkeye thanks for tagging! Looking at the PR right now

phacks commented on src/gatsby-node.js:
```diff
@@ -1,6 +1,46 @@
 const fetch = require("node-fetch")
 const queryString = require("query-string")
 
+exports.createSchemaCustomization = ({ actions }) => {
```
Thanks for the PR!

As mentioned in the [Gatsby docs](https://www.gatsbyjs.org/docs/schema-customization/):

> Actions to customize Gatsby’s schema generation are made available in the createSchemaCustomization (available in Gatsby v2.12 and above), and sourcesNodes APIs.

I would then suggest mentioning that the Gatsby Source Meetup plugin requires `gatsby >= 2.12.0` in the README and bump the peerDependency requirement in the `package.json`:

https://github.com/phacks/gatsby-source-meetup/blob/e7660c6cc6c4748f488c8d30725ed775d58c2406/package.json#L34-L36

pethersonmoreno replied:
Thanks @phacks .
Suggestion applied.

pethersonmoreno commented:
@phacks , I completed the changes, could you validate the changes?

phacks commented:
Hi @pethersonmoreno, thanks for updating the PR!

I’ve tried running it locally and it seemed to work as intended, however I do not have a “failing” test case to test it against. Do you have a meetup group in mind for which this bug can be reproduced (and check that the fix works as intended)?

phacks commented:
Hi! Can confirm that this feature works as intended and it fixes the issues with the `Nodeschool-Campinas` meetup group you gave as example, thanks for your help!

However, it appears that there are some missing fields in the schema, which can break other meetups (such as `JAMstack-Paris`). Tried running your branch locally, and got the following error: `Cannot query field "local_date" on type "MeetupEventVenue".` 

I guess we’ll have to include every possible field in the GraphQL schema mentioned in [the docs](https://www.meetup.com/fr-FR/meetup_api/docs/:urlname/events/?uri=%2Fmeetup_api%2Fdocs%2F%3Aurlname%2Fevents%2F#list), so that this will not be a breaking change for some users. Would you be willing to do that? I can take it from here if you’d rather.

pethersonmoreno commented:
Hi @phacks , I can't do it now, if you want to change it you can too. If not until tomorrow morning I will do the changes requested.

pethersonmoreno commented:
> Hi! Can confirm that this feature works as intended and it fixes the issues with the `Nodeschool-Campinas` meetup group you gave as example, thanks for your help!
> 
> However, it appears that there are some missing fields in the schema, which can break other meetups (such as `JAMstack-Paris`). Tried running your branch locally, and got the following error: `Cannot query field "local_date" on type "MeetupEventVenue".`
> 
> I guess we’ll have to include every possible field in the GraphQL schema mentioned in [the docs](https://www.meetup.com/fr-FR/meetup_api/docs/:urlname/events/?uri=%2Fmeetup_api%2Fdocs%2F%3Aurlname%2Fevents%2F#list), so that this will not be a breaking change for some users. Would you be willing to do that? I can take it from here if you’d rather.

Included every possible field in GraphQL schema mentioned in docs (only event schema):

- https://www.meetup.com/pt-BR/meetup_api/docs/:urlname/events/?uri=%2Ffr-FR%2Fmeetup_api%2Fdocs%2F%3Aurlname%2Fevents%2F#list

But about the field `local_date` on type "MeetupEventVenue", really doesn't exist it there, but exist it in MeetupEvent.

I've already updated the branch with your master updates and fixed another error (caused by my other Pull request), updated the tests to match 100% coverage.

I think that we can do more tests to verify the graphql working with the changes, I only applied Unit Tests, we need Component Tests too, but It can come in another Pull request.

Now, I think it is ok to Merge. please @phacks validate the new changes.
